bin/load_env_secrets.sh

```diff
@@ -69,6 +69,9 @@ if [[ -z $SERVICE_ACCOUNT_PATH ]] && [[ -z $VAULT_SECRET_PATH ]] ; then
   exit 1
 fi
 
+#  clear this evironment variable just in case this terminal was used for local development
+export NOT_DOCKERIZED=
```

Comment:
[`unset`](https://stackoverflow.com/a/6877747) seems like best practice for deleting environment variables.
```suggestion
unset NOT_DOCKERIZED
```